drivers/mhz19/include/mhz19_internals.h

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2018 Koen Zandberg <koen@bergzand.net>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_mhz19
+ * @{
+ * @file
+ * @brief       Internal addresses, registers, constants for the MH-Z19 CO2 sensor
+ *
+ * @author      Koen Zandberg <koen@bergzand.net>
+ */
+
+#ifndef MHZ19_INTERNALS_H
+#define MHZ19_INTERNALS_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    MH-Z19 Baud rate
+ *
+ * Fixed at 9600 by design.
+ */
+#define MHZ19_UART_BAUDRATE             9600
```

Comment:
I think some whitespace can be removed. I dont think you have to align all defines in the file at the same level, but per block.

Reply:
Removed some whitespace here and below this line